lib/uploadcare/client/multipart_upload/chunks_client.rb

```diff
@@ -18,9 +18,10 @@ class ChunksClient < ApiStruct::Client
         # @param links [Array] of strings; by default list of Amazon storage urls
         def upload_chunks(object, links)
```

Comment:
I don't like that when we call `client.upload_chunks`, we actually don't use `client` in any way.
Can we make this method class-level?

Reply:
And than maybe add
```ruby
        def upload_chunks(object, links)
          self.class.upload_chunks(object, links)
        end
```